test/test-cases/scale/saic/test_sai_vnet_route_bidirectional.py

```diff
@@ -0,0 +1,219 @@
+        results = [*dpu.process_commands(setup_commands)]
+        print("\n======= SAI commands RETURN values =======")
+        pprint(results)
+        assert all(results), 'Create error'
+
+
+    @pytest.mark.snappi
+    def test_vnet_route_packet_bidirectional_forwarding_with_route_match(self, dpu, dataplane):
+        """Verify packet forwarding with route match"""
+ 
+        """
+        Verify same config with high-rate traffic.
+        packets_per_flow=10 means that each possible packet path will be verified using 10 packet.
+        NOTE: For BMv2 we keep here PPS limitation
+        """
+        dataplane.set_config()
+
+        inner_pkt_one = simple_udp_packet(eth_dst = "02:02:02:02:02:02",
+                                      eth_src = "00:cc:cc:cc:00:00",
+                                      ip_dst  = "10.1.2.50",
+                                      ip_src  = "10.1.1.10")
+        vxlan_pkt_one = simple_vxlan_packet(eth_dst         = "00:00:02:03:04:05",
+                                        eth_src         = "00:00:05:06:06:06",
+                                        ip_dst          = "172.16.1.100",
+                                        ip_src          = "172.16.1.1",
+                                        udp_sport       = 11639,
+                                        with_udp_chksum = False,
+                                        vxlan_vni       = 100,
+                                        inner_frame     = inner_pkt_one)
+        
+        # send_packet(dataplane, 1, vxlan_pkt, 111)
+        inner_pkt_two = simple_udp_packet(eth_dst = "00:00:00:09:03:14",
+                                      eth_src = "00:0a:04:06:06:06",
+                                      ip_dst  = "172.19.1.100",
+                                      ip_src  = "172.19.1.1")
+        vxlan_pkt_two = simple_vxlan_packet(eth_dst         = "00:0b:05:06:06:06",
+                                        eth_src         = "00:0a:05:06:06:06",
+                                        ip_dst          = "10.10.2.20",
+                                        ip_src          = "10.10.2.10",
+                                        udp_sport       = 11638,
+                                        with_udp_chksum = False,
+                                        vxlan_vni       = 60,
+                                        inner_frame     = inner_pkt_two)
+        
+        send_packet(dataplane, 0, vxlan_pkt_one, 111)
+        send_packet(dataplane, 1, vxlan_pkt_two, 777)
+
+        time.sleep(10)
+        rows = dataplane.get_all_stats()
+        print("{}".format(rows[0].name))
+        print("Transmission_Frames : {}".format(rows[0].frames_tx))
+        print("Recieved_Frames : {}".format(rows[0].frames_rx))
+        print("{}".format(rows[1].name))
+        print("--------------------------")
+        print("Transmission_Frames : {}".format(rows[1].frames_tx))
+        print("Recieved_Frames : {}".format(rows[1].frames_rx))
+
```

Comment:
Where is the test assertion that the expected number of packets were received through the route?